packages/react/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { forwardRef } from 'react';
+import type { ForwardedRef, ReactNode, SelectHTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import { InputWrapper } from '../_InputWrapper';
+
+export type NativeSelectProps = {
+  /** Instances of `option` */
+  children?: ReactNode;
+
+  /** Specifies whether the select box is disabled. */
+  disabled?: boolean;
+
+  /** Specifies whether the selected value is valid. */
+  isValid?: boolean;
+
+  /** Label that appears over the select box. */
+  label?: string;
+
+  /** Set to true to enable multiple selection. */
+  multiple?: boolean;
+
+  /** The ID of the `select` element. This will be generated if not provided. */
+  id?: string;
+
+  /** Defines the number of visible options. */
+  size?: number;
+
+  /** The class name for the wrapper element. */
+  wrapperClassName?: string;
+
+  /** The ID of the wrapper element. */
+  wrapperId?: string;
```

Comment:
Can we find a better way of providing "container" props for users? Not only are we missing `style` prop here but duplicating props prefix with "wrapper" does not feel like it might scale to well, and if we start with this pattern here, we would also have to do it for our other components.

In the past we have solved this two ways
- `classname` and `style` are always passed to container. Users can use selector to style encompassed element if needed.
- Do it the other way with proving an `selectProps` prop which acts like a "...rest" for the core element. Inspired from [MUI](https://mui.com/material-ui/api/text-field/)

Reply:
I agree that we should take a closer look to make a more informed decision on this. I'll remove these props for now.

Reply:
I have removed them now, but I kept the `className` on the `select` element since this is the way it is currently done on the other components.

Reply:
Hmm, has this worked ok? General rule of thumb has been that `className` & `style` goes on the outermost element as this is what people need for positioning the component it in their apps. If its adding margins, setting flex props, grid-template-areas etc...

Reply:
I'm not sure how much this is actually used right now. Anyways, I agree, it makes more sense to set these attriubutes on the root element. I'll create an issue for it.

Reply:
Suggest we can change this now for this component, since its still in development. We'll have separate "improvements"  issues for the other component in the library in the not so long future :)